site/ja/blog/origin-trials/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Chromeのオリジントライアル入門
+subhead: オリジントライアルは、新規の、または実験的なウェブプラットフォームの機能をテストするための方法です。また、その機能が全てのユーザ向けに利用可能になる前に、その機能の使いやすさ、実用性、そして有効性についてウェブ標準コミュニティにフィードバックする方法でもあります。
+authors:
+  - samdutton
+date: 2020-06-22
+updated: 2021-09-01
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/KeaVCdXHWzrI35QRvsZL.jpg
+alt: Pipette with purple liquid
+tags:
+  - origin-trials
+---
+
```

Comment:
Put the translator here @yoichiro

Reply:
@chybie Thank you for the suggestion! I will put the expression based on the What's new in DevTools article on the top of this article.

Reply:
@chybie I put it at 104959b.